packages/kubernetes-1.24/kubelet-config

```diff
@@ -147,6 +147,13 @@ serverTLSBootstrap: {{settings.kubernetes.server-tls-bootstrap}}
 {{/if}}
 tlsCipherSuites:
 - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+- TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+- TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
+- TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+- TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
+- TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+- TLS_RSA_WITH_AES_256_GCM_SHA384
+- TLS_RSA_WITH_AES_128_GCM_SHA256
```

Comment:
From [the Go 1.22 release notes](https://tip.golang.org/doc/go1.22#minor_library_changes)
> By default, cipher suites without ECDHE support are no longer offered by either clients or servers during pre-TLS 1.3 handshakes. This change can be reverted with the tlsrsakex=1 GODEBUG setting.

So I expect these last two can't actually be negotiated unless we took special steps to allow it, but that should be fine.